data/week1/3-printing.md

```diff
@@ -24,19 +26,19 @@ Hello World!
 
 </sample-output>
 
-The same command can be used to print other text as well. For example, we might want to say goodbye as well. We would then again use the `System.out.println` command, now putting a different value between the brackets. For example, we could use the command as follows:
+The same command can be used to print other text as well. For example, to print ``Goodbye!``, we would again use the `System.out.println` command, but now putting a different value between the brackets. For example, we could use the command as follows:
 ``` Java
 System.out.println("Goodbye!")
 ```
-This then leads to the below output being printed to the console.
+This leads to the below output being printed to the console.
 
 <sample-output>
 
 Goodbye!
 
 </sample-output>
 
-The input that we give between brackets is referred to as an `argument` of the function. In the first case, we gave as argument to the function `System.out.println` the text ``"Hello World!"``, while in the second case our argument was ``"Goodbye!"``. Note that we can use any argument to this function, as long as it is composed of text, as indicated by the quotation marks `""` around the text that we want to print.
+The text within the brackets of a function (in this case `System.out.println`) is referred to as an `argument` and can be any text enclosed in quotation marks `""`. Our first example used `"Hello World!"` as an argument, while the second used `"Goodbye!"`.
```

Comment:
Kan de suggestie wekken dat elk argument tussen aanhalingstekens moet. Misschien zo:

The text within the brackets of a function (in this case `System.out.println`) is referred to as an `argument` and can be any text. Our first example used `"Hello World!"` as an argument, while the second used `"Goodbye!"`. But we can use any text, as long as long as it is enclosed in quotation marks `""`.